docs/en_us/platform_api/source/course_structure/overview.rst

```diff
@@ -0,0 +1,36 @@
+.. _EdX Platform Course Structure API Overview:
+
+################################################
+EdX Platform Course Structure API Overview
+################################################
+
+Use the edX Platform Course Structure API to view course details, including the
+blocks in the course and the course grading policy.
+
+The edX Platform Course Structure API uses Representational State Transfer
+(REST) design principles and supports the JavaScript Object Notation (JSON)
+data-interchange format. The REST API is simple and lightweight.
+
+You can use the edX Platform Course Structure API for web, desktop, and mobile
+applications.
+
+********************************************
```

Comment:
We can give a more nuanced explanation of versions later, but for now in the overview page, can we group the listing by "Supported" (Enrollment, Mobile) and "Experimental" (the other stuff)?